server/controllers/projects/post.js

```diff
@@ -1 +1,26 @@
-exports.post = (req, res) => res.send('single project new');
+const joi = require('joi');
+
+const { insertProject, insertProjectUsers } = require('./../../database/queries/insertProject');
+
+exports.post = (request, response) => {
+  const schema = joi.object().keys({
+    name: joi.string(),
+    dsescription: joi.string(),
+  });
+  const { name, dsescription, row } = request.body;
+  const result = joi.validate({ name, dsescription }, schema);
+
+  if (result.error === null) {
+    insertProject(name, dsescription)
+      .then((res) => {
+        const newProjectID = res.rows[0].id;
+        return insertProjectUsers(newProjectID, row);
+      })
+      .then(() => {
+        response.send({ error: null, data: [{ msg: 'add new project successfully' }] });
```

Comment:
here you should return the new project, not just successful msg, in case they wanted to add it to the table without refreshing the page